### DISCUSSION TITLE
showing the final model (only)

### DISCUSSION THREAD

**Discussion by ralikwen** [lindeloev/mcp#134] [Q&A] [answered]
How can I now show the final model with a different colored line?
Maybe it could be an option to show the final model with another color, or only show the final model.

**Comment by lindeloev** [comment, marked as answer]
Could you say more about what you mean by "the final model"? Is it the model based on the mean values in `summary()`? One way to approximate that would be to plot the median (dashed red line):

```r
fit = mcp_example("demo", sample = TRUE)$fit
plot(fit, q_fit = 0.5)  # 50% quantile, i.e., the median
```

![image](https://user-images.githubusercontent.com/1439417/142990008-9c90df2a-f436-41ba-9de7-355f5bd4bff2.png)

You can disable everything else to show just the model:

```r
plot(fit, q_fit = 0.5, lines = 0, geom_data = FALSE, cp_dens = FALSE)
```
![image](https://user-images.githubusercontent.com/1439417/142990421-eba50cdc-3a70-4b65-8673-742d82ec6f19.png)

**Comment by ralikwen** [comment]
This was exactly what I was looking for.
Thanks.
